CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+### ⚠️ Breaking Changes from `4.0-beta.8`
+- `ChatMessageLayoutOptionsResolver.optionsForMessage` has a new parameter: `appearance`. Please add this new param to your overrides. [#1304](https://github.com/GetStream/stream-chat-swift/issues/1304)
```

Comment:
perhaps we add an example on how to resolve this? sounds like in all cases you can fix this by passing `self.appearance` or `Appearance.default`